apps/storefront/lib/useUser.ts

```diff
@@ -1,7 +1,7 @@
 import { useUserQuery } from "@/saleor/api";
 
 export const useUser = () => {
-  const { data, loading } = useUserQuery();
+  const { data, loading } = useUserQuery({ fetchPolicy: "no-cache" });
```

Comment:
What about other queries that may depend on the user being logged in, for example `checkout`?

Reply:
@mmiszy changed to `cache-and-network`